openpower/package/ppe42-binutils/Config.in

```diff
@@ -1,6 +1,6 @@
 config BR2_PACKAGE_HOST_PPE42_BINUTILS
         bool "ppe42-binutils"
-        default y if (BR2_OPENPOWER_PLATFORM && BR2_OPENPOWER_POWER9)
+        default y if BR2_PACKAGE_HOST_PPE42_GCC
```

Comment:
Should this change be separate? Why are we changing the default selection?

Reply:
- I moved the `default y if (BR2_OPENPOWER_POWER9)` to the ppe42-toolchain virtual package definition (https://github.com/open-power/op-build/pull/3625/files#diff-dc9e0d3bb515095fc9adb28b93f03622R4)
- `BR2_OPENPOWER_PLATFORM` was already protecting `$BR2_EXTERNAL_OP_BUILD_PATH/package/Config.in` which was including `openpower/package/ppe42-binutils/Config.in` to begin with, so it was a noop. But note that I'm now including `openpower/package/ppe42-binutils/Config.in` as part of the _OpenPower Toolchain_ menu which is NOT protected by `BR2_OPENPOWER_PLATFORM` - reason is to hopefully make it easier to disable all target packages (by unsetting `BR2_OPENPOWER_PLATFORM`) while still allowing the openpower toolchain packages to be built.
- `default y if BR2_PACKAGE_HOST_PPE42_GCC` could be a the reminiscence of a hack that I may need to do some more testing on. Originally, SBE, HCODE and OCC were depending on _host-ppe42-gcc_  in their .mk files, and _ppe42-gcc_ depends on _ppe42-binutils_. When converting to use virtual-packages, somewhere along the way buildroot started complaining about having dependencies on .mk files that were not selected by the Config.in file. I'll do some more testing on this